src/widget.js

```diff
@@ -231,6 +231,51 @@ function pageVisited(change) {
 	});
 }
 
+function initialize(params) {
+	for (const method in params) {
+		switch(method) {
+			case 'customFied':
+				setCustomField(params[method]);
+				continue;
+			case 'theme':
+				setTheme(params[method]);
+				continue;
+			case 'department':
+				setDepartment(params[method]);
+				continue;
+			case 'guestToken':
+				setGuestToken(params[method]);
+				continue;
+			case 'guestName':
+				setGuestName(params[method]);
+				continue;
+			case 'guestEmail':
+				setGuestEmail(params[method]);
+				continue;
+			case 'registerGuest':
+				registerGuest(params[method]);
+				continue;
+			case 'language':
+				setLanguage(params[method]);
+				continue;
+			case 'hideWidget':
+				hideWidget();
+				continue;
+			case 'showWidget':
```

Comment:
Let's skip the methods below, okay? In this first implementation let's focus on the `props` only, okay?

Reply:
:+1: